src/routes/feedback/components/FeedbackModal.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import React from 'react'
+import FeedbackBox from './FeedbackBox'
+import CityModel from '../../../modules/endpoint/models/CityModel'
+import styled from 'styled-components'
+import CleanLink from '../../../modules/common/components/CleanLink'
+import type { LocationState } from 'redux-first-router'
+import { goToFeedback } from '../../../modules/app/routes/feedback'
+
+const Overlay = styled(CleanLink)`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: ${props => props.theme.colors.textSecondaryColor};
+  opacity: 0.9;
+`
+
+const ModalContainer = styled.div`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+  display: ${props => props.isOpen ? 'flex' : 'none'};
+  align-items: center;
+  justify-content: center;
+`
+
+const FeedbackContainer = styled.div`
+  position: relative;
+  z-index: 3;
```

Comment:
This is also unnecessary, if you apply the suggestion below...
(Reduce z-indices because they're strange^^)